pwiz_tools/Skyline/TestData/CommandLineTest.cs

```diff
@@ -289,6 +289,40 @@ public void ConsoleAddFastaTest()
             AssertEx.IsDocumentState(doc, 0, 2, 7, 7, 49);
         }
 
+        [TestMethod]
+        public void ConsoleNewDocumentTest()
+        {
+            TestFilesDir = new TestFilesDir(TestContext, ZIP_FILE);
+            string docPath = TestFilesDir.GetTestPath("BSA_Protea_label_free_20100323_meth3_multi.sky");
+            string outPath = TestFilesDir.GetTestPath("AddFasta_Out.sky");
+            string fastaPath = TestFilesDir.GetTestPath("sample.fasta");
+
+            // arguments that would normally be quoted on the command-line shouldn't be quoted here
+            var settings = new[]
+            {
+                "--new=" + docPath,
+                "--full-scan-precursor-isotopes=Count",
+                "--full-scan-acquisition-method=DDA",
+                "--tran-precursor-ion-charges=2,3,4",
+                "--tran-product-ion-charges=1,2",
+                "--tran-product-start-ion=ion 1",
+                "--tran-product-end-ion=last ion - 1"

```

Comment:
I probably mis-copied it from the GUI tutorial. Latest update uses:
                "--tran-product-start-ion=" + TransitionFilter.StartFragmentFinder.ION_3.Label,
                "--tran-product-end-ion=" + TransitionFilter.EndFragmentFinder.LAST_ION.Label,

Reply:
Oh, I didn't notice you were commenting on the settings for the functional test, not the tutorial. I'm not sure why you're commenting on the usefulness of the value in the context of a functional test though because the point is just to make sure the argument changes the document setting the way it's supposed to.